setup.py

```diff
@@ -7,15 +7,15 @@
 # NOTE: When updating botocore make sure to update awscli/boto3 versions below
 install_requires = [
     # pegged to also match items in `extras_require`
-    'botocore>=1.34.70,<1.34.107',
+    'botocore>=1.34.107,<1.34.132',
     'aiohttp>=3.9.2,<4.0.0',
     'wrapt>=1.10.10, <2.0.0',
     'aioitertools>=0.5.1,<1.0.0',
 ]
 
 extras_require = {
-    'awscli': ['awscli>=1.32.70,<1.32.107'],
```

Comment:
Since there appear to be no relevant upstream changes, then lower bound can be kept at 1.32.70 and 1.34.70 respectively.